WordPress/Classes/ViewRelated/Stats/Shared Views/StatsTotalRow.swift

```diff
@@ -41,11 +43,17 @@ struct StatsTotalRowData {
         self.disclosureURL = disclosureURL
         self.childRows = childRows
         self.statSection = statSection
+        self.isReferrerSpam = isReferrerSpam
     }
 
     var hasIcon: Bool {
         return self.icon != nil || self.socialIconURL != nil || self.userIconURL != nil
     }
+
+    var canMarkReferrerAsSpam: Bool {
+        disclosureURL != nil && disclosureURL!.absoluteString.contains(name) ||
```

Comment:
Yes, I agree, I'll add tests for this. Also, I thought force unwrap is okay in this case since with this short circuit evaluation crash can never happen (since first check is `!= nil`). But I can remove the `!` operator.

Reply:
>Also, I thought force unwrap is okay in this case since with this short circuit evaluation crash can never happen (since first check is != nil). But I can remove the ! operator

While this is an appropriate use we like to avoid using force unwraps in the codebase.